autoload/neoterm/term.vim

```diff
@@ -31,7 +31,11 @@ function! s:term.normal(cmd)
 endfunction
 
 function! s:term.exec(command)
-  call l:self.termsend(l:self.termid, a:command)
+  if !empty(g:neoterm_command_prefix)
```

Comment:
can you include only the `racket` code, please?

Reply:
Sorry, meant to only include the d5d0e4e commit in this PR